### PR TITLE
Various script changes to accommodate shared framework.

### PR DESCRIPTION
**Pull request by shhsu**
Some changes made a while ago. Need to merge in

### CODE REVIEW

**Comment by shhsu** [comment]
These are some script changes requested by @SajayAntony a while ago. Forgot to merge in afterwards.

I have rebased and please review

Also include @sivagms

File: test/ColdStart/InstallDotnet.sh [hunk 1]
```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-## This script requires sudo
-
-sudo sh -c 'echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
```
Comment (DavidObando) [inline test/ColdStart/InstallDotnet.sh]: Is this script Ubuntu-specific?

**Comment by DavidObando** [comment]
:shipit:

**Comment by DavidObando** [comment]
Does this PR override #131?

**Comment by shhsu** [comment]
Correct, I will be removing that PR now

**Pull request merged**